src/content/docs/tutorial-errors/reduce-common-errors.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: Reduce common errors
+metaDescription: Learn how to manage a high number of errors in your app.   
+---
+
+So far, you've learned how to scope out a service outage and get the error in front of the correct teams. Error analysis doesn't end after the solution is deployed, however. To return to our flooded yard metaphor, you've only diagnosed the leaking pipe as the direct cause of the flood, then ordered a patch to fix it. You aren't done, though. At this point, you should ask yourself: What caused the leaking pipe to begin with? 
+
+## Objectives [#objectives]
+
+This tutorial helps you understand the difference between direct cause and root cause so you can reduce common errors over time. By the end of this tutorial, you will be able to:
+
+* Route errors to the correct teams
+* Diagnose the root cause of an incident
+
+## Route errors to the correct teams [#route]
+
+To send errors to other teams, you need to return to the **Errors** summary page. If you don't own a component or functionality, you can use this page to send grouped errors to the correct teams.  
+
+    <img
+        title="Overview errors affecting your services"
+        alt="A screenshot showing an app with many errors"
+        src={apmTriageError}
+    />  
+
+In the previous tutorial docs, we covered how to quickly respond to a critical error. But even without a service outage, we recommend that you return to the **Errors** page for at least 10 minutes on a weekly cadence. 
+
+* Imagine a patch that was deployed 48 hours ago on a service. During your weekly triage, you notice a new error has appeared in your errors inbox. An alert hasn't fired because the error occurence hasn't met your alert threshold, but you do notice an increase in error incidences. 
+* You route this error to the correct team even though there's a chance it's an expected error.
+* Because the deployment was recently introduced, the owning team still has context to evaluate the scope of the problem. In short, the person who made the deployment is still familiar enough to reproduce the problem, something that might not be possible in several months time. 
+
+When you spend time maintaining your errors, you can more easily keep track of what's actually expected or what's anomalous. You become more independent when you're proactive instead of reactive to external notifications. 
+
+If that same team responds that that error is expected, you can then close out these errors by selecting **Ignore**. We recommend resolving expected errors as much as you can, however. Ignoring an error group doesn't prevent New Relic from reporting the error in the future, which contributes to your data ingest.
```

Comment:
This paragraph might also explain what resolving an error means and how if it is marked as resolved and then reoccurs at a later point, it will be marked as a regression.